developer-hub/src/polkadot_sdk/polkadot.rs

```diff
@@ -0,0 +1,81 @@
+//! # Polkadot
+//!
+//! Implementation of the Polkadot node/host in Rust.
+//!
+//! ## Getting Involved
+//!
+//! - [Polkadot Forum](https://forum.polkadot.network/)
+//! - [Polkadot Parachains](https://parachains.info/)
+//! - [Polkadot (multi-chain) Explorer](https://subscan.io/)
```

Comment:
There are many explorers out there, should we mention more of them? statescan for example